db/migrations/20170821164207_add_name_to_movies.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+exports.up = function (Knex, Promise) {
+  return Knex.schema.table('movies', (table) => {
+    table.text('name');
+  })
+  .then(() => {
+    return Knex.raw('ALTER TABLE movies ALTER COLUMN title DROP NOT NULL');
```

Comment:
I'm not sure this is the place you want to `DROP NOT NULL`. Ideally, you keep it until you just drop the column, and then just double write to both columns.